GAME/CAMERA.C

```diff
@@ -17,10 +17,6 @@
 	#include <math.h>
 #endif
 
-#if PSX_VERSION
```

Comment:
This is specifically needed for the PSX version hence why it's wrapped in if PSX_VERSION. Removing this will break PSX.